db/seeds.rb

```diff
@@ -6,57 +6,57 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-require 'faker'
-
-(1..20).each do |id|
-  User.create!(
-      id: id, 
-      name: Faker::Name.name,
-      photo: Faker::Lorem.sentence,
-      bio: Faker::Lorem.paragraph(sentence_count: 15, supplemental: true),
-      post_counter: 0,
-  )
-end
-
-(1..100).each do |id|
-  Post.create!(
-      id: id,
-      author_id: rand(1..20),
-      title: Faker::Book.title,
-      text: Faker::Lorem.paragraph(sentence_count: 5, supplemental: true),
-      comments_counter: 0,
-      likes_counter: 0,
-  )
-end
-
-(1..100).each do |id|
-  Comment.create!(
-      id: id,
-      post_id: rand(1..100),
-      author_id: rand(1..20),
-      text: Faker::Lorem.paragraph,
-  )
-end
-
-(1..100).each do |id|
-  Like.create!(
-      id: id,
-      author_id: rand(1..20),
-      post_id: rand(1..100),
-  )
-end
-
-User.all.each do |u|
-  u.post_counter = u.posts.count
-  u.save
-end
-
-Post.all.each do |p|
-  p.comments_counter = p.comments.count
-  p.likes_counter = p.likes.count
-  p.save
-end
-
-ActiveRecord::Base.connection.tables.each do |t|
-ActiveRecord::Base.connection.reset_pk_sequence!(t)
-end
+# require 'faker'
+
+# (1..20).each do |id|
+#   User.create!(
+#       id: id, 
+#       name: Faker::Name.name,
+#       photo: Faker::Lorem.sentence,
+#       bio: Faker::Lorem.paragraph(sentence_count: 15, supplemental: true),
+#       post_counter: 0,
+#   )
+# end
```

Comment:
- [ ] [OPTIONAL] You should rather delete files or code fragments that you do not need rather than commenting them out.